metricbeat/module/mysql/galera-status/data.go

```diff
@@ -0,0 +1,144 @@
+package galerastatus
+
+import (
+	"fmt"
+
+	"github.com/elastic/beats/libbeat/common"
+	s "github.com/elastic/beats/libbeat/common/schema"
+	c "github.com/elastic/beats/libbeat/common/schema/mapstrstr"
+)
+
+var (
+	// Schema for mapping all Galera-Status-Variables
+	schemaFull = s.Schema{
+		"apply": s.Object{
+			"oooe": c.Float("wsrep_apply_oooe"),
+			"oool": c.Float("wsrep_apply_oool"),
+			"window": c.Float("wsrep_apply_window"),
+		},
+		"cert": s.Object{
+			"deps_distance": c.Float("wsrep_cert_deps_distance"),
+			"index_size": c.Int("wsrep_cert_index_size"),
+			"interval": c.Float("wsrep_cert_interval"),
+		},
+		"cluster": s.Object{
+			"conf_id": c.Int("wsrep_cluster_conf_id"),
+			"size": c.Int("wsrep_cluster_size"),
+			"status": c.Str("wsrep_cluster_status"),
+		},
+		"commit": s.Object{
+			"oooe": c.Float("wsrep_commit_oooe"),
+			"window": c.Float("wsrep_commit_window"),
+		},
+		"connected": c.Str("wsrep_connected"),
+		"evs": s.Object{
+			"evict": c.Str("wsrep_evs_evict_list"),
+			"state": c.Str("wsrep_evs_state"),
+		},
+		"flow_ctl": s.Object{
+			"paused": c.Float("wsrep_flow_control_paused"),
+			"paused_ns": c.Int("wsrep_flow_control_paused_ns"),
```

Comment:
These metrics are quite important in galera, they probably deserve to be also in the small metric set :slightly_smiling_face:

Reply:
done. =)